Bleedify/BleedifyModels/ModelsEF/Extensions/Personal.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BleedifyModels.ModelsEF
+{
+    partial class Personal : IHasID<int>
```

Comment:
Am specificat in document ca Personal, Donator si Medic mostenesc Utilizator, te rog sa te mai uiti o data pe document si sa modifici.